[NCP-3228] Added report.export.group.csv.text

hwaeunseo opened the pull request:
In the ngportal-language.babel file, testsuites previously added by junghoPark were also added.

burtonquantil commented:
The text change looks OK to me though I wonder why ngportal-languange.babel was modified in this pull request. It is not updated all the time. For example, @jungho73 's updates for NCP-1440 didn't require changes to this file.

hwaeunseo commented:
@burtonquantil 
When adding new text in babel Edit, it is also added to the ngportal-language.babel file and modified.
When should the babel file be modified? Are there any rules?
If the Babel file should not be modified even when adding a new text, I will not modify the Babel file from next time.

burtonquantil reviewed (approved):
The text change looks OK to me though I wonder why ngportal-languange.babel was updated in this case as well though it doesn't seem to be checked in by everybody in each pull request.

burtonquantil commented:
@hwaeunseo  I do not know how BabelEdit determines when that file is updated. I just noticed that some, but not all, pull requests update it, so I was wondering why.

hwaeunseo commented:
@burtonquantil 
I updated the babel file when I added a new text before
If you decide on the update rules for that babel file, I'll follow them
Thank you.
